api/src/test/java/org/apache/cloudstack/api/command/admin/resource/ListCapacityCmdTest.java

```diff
@@ -0,0 +1,34 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package org.apache.cloudstack.api.command.admin.resource;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.springframework.test.util.ReflectionTestUtils;
+
+public class ListCapacityCmdTest {
```

Comment:
@DaanHoogland do we need to add this to make this test considered for coverage?
```java
@RunWith(MockitoJUnitRunner.class)
```

Reply:
@vishesh92 if this helps then will add it to this and other added test files. I didn't know. Thanks.

Reply:
only if you don't go for the default runner as i always understood it (which means not much btw)